### ISSUE TITLE
State of traffic light icons difficult to see when using Libation's dark theme setting

### ISSUE THREAD

**Issue by mattalexx** [rmcrackan/Libation#702]
**Describe the bug**
In Libation, I have the "Important Settings > Theme" setting set to "Dark". In the main grid, the traffic light icons "Liberate" column are difficult to see.

**To Reproduce**

1. Install Linux versions of KDE Plasma and Libation.
4. Open Libation.
3. Open "Important Settings > Theme" and choose "Dark".
7. Notice that red, yellow, and green traffic lights look very similar to each other.

**Expected behavior**

To be able to tell the "Liberated" state by a quick glance over the column.

**Screenshots**

![Screenshot](https://i.imgur.com/bBWkylM.png)

**Platform**

Arch Linux

```
% grep -v _URL /etc/os-release
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
LOGO=archlinux-logo
% uname -a
Linux m 6.4.8-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 03 Aug 2023 16:02:01 +0000 x86_64 GNU/Linux
% plasmashell --version
plasmashell 5.27.7
% kwin_x11 --version
kwin 5.27.7
% libationcli version
Libation Chardonnay v10.6.5
```

**Log Files**
n/a

**Comment by Mbucari** [comment]
Hey, I chose those colors!

I'm not a graphics guy by any stretch, so if you'd like to play around with a photo editor and come up with something better, we'll most likely agree to it and implement it.

**Comment by CLHatch** [comment]
> Hey, I chose those colors!
> 
> I'm not a graphics guy by any stretch, so if you'd like to play around with a photo editor and come up with something better, we'll most likely agree to it and implement it.

I noticed they seemed a bit dark myself.  Maybe a more "vibrant" set of red/yellow/green colors would stand out more?

**Comment by Mbucari** [comment]
I spent way too much time fiddling around with those colors to come up with what you see. I'm happy enough with it, so I'm not going to spend any more time screwing around with colors. But if anyone else wants to, please go for it.

**Comment by CLHatch** [comment]
> I spent way too much time fiddling around with those colors to come up with what you see. I'm happy enough with it, so I'm not going to spend any more time screwing around with colors. But if anyone else wants to, please go for it.

I get you there. :-D  I've played for hours on end coming up with working colors for things myself.

**Comment by mattalexx** [comment]
> Hey, I chose those colors!
> 
> I'm not a graphics guy by any stretch, so if you'd like to play around with a photo editor and come up with something better, we'll most likely agree to it and implement it.

Thanks for your great work!

Please forgive me, I have a bad habit of filing nitpicky bugs and feature requests when I'm playing with new software that I'm enjoying, like Libation. But beggars can't be choosers. So I do subsequently try to balance them out with PRs if I have the time.

I haven't worked with C# for years and I would like to see what .NET Core is like now (it was not very mature when I left the job where I used .NET). C# is such a great language though, especially when you have Visual Studio to use (which I won't). Another thing that was too new at the time was JetBrains Rider, and I assume/hope that's changed by now.

**Comment by Mbucari** [comment]
>  I have a bad habit of filing nitpicky bugs and feature requests when I'm playing with new software that I'm enjoying, like Libation.

Keep em coming! Most of the new features aren't dreamed up by us but instead come from the community. Dark mode only exists because someone asked for it.

-----------

Rider is a great option. I use it when I'm working on Linux and Mac (Visual Studio on macOS is a joke), but I primarily work on Windows with VS Community, and it suits me just fine.

------------
I put together a mock SVG of the stoplight icon for you or @CLHatch to fiddle with. Post your best color scheme. We can make it sort of a competition. Winner gets 1 free month of Libation Premium. It's just like Libation Basic but with bragging rights.

![test](https://github.com/rmcrackan/Libation/assets/37587114/79425caa-f4bf-4ecf-bf2b-c6cd3b2481d3)

**Comment by Mbucari** [comment]
Fixed in version [v11.0.0](https://github.com/rmcrackan/Libation/releases/tag/v11.0.0)
